examples/examples_test.go

```diff
@@ -389,6 +389,15 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"javaweb":   &api.Pod{},
 			"javaweb-2": &api.Pod{},
 		},
+		"../examples/job/expansions": {
+			//"job":   &api.Pod{},
+		},
+		"../examples/job/work-queue-1": {
+			//"job":   &api.Pod{},
+		},
+		"../examples/job/work-queue-2": {
+			//"job":   &api.Pod{},
+		},
```

Comment:
Unfortunately the job-template needs to be skipped since it's a template and it fails validation. This in turn leads to not checking its syntax, the same applies to job-expand.py.